Loop/Managers/DeviceDataManager.swift

```diff
@@ -391,10 +410,58 @@ final class DeviceDataManager: CarbStoreDelegate, CarbStoreSyncDelegate, DoseSto
                     nsPumpStatus = nil
                 }
                 self.nightscoutDataManager.uploadDeviceStatus(nsPumpStatus)
+                completion?()
             }
         }
     }
 
+    private func updatePumpCGMSensorStatus(_ events: [TimestampedGlucoseEvent]) -> Void {
```

Comment:
Agree about the `// MARK: Enlite` being a cosmetic change, but it would be nice here. This file is the high level thin layer of state above all the device data that we manage; as long as the code here is more about how that data fits/integrates into Loop, and not just managing device details, then I think it's ok.